pods-framework/pods
# Abstract DFV API and start abstracting config handling

## Description

Abstracts the DFV API from the main DFV script so it can be better structured.

Previously, everything was stuffed directly in `pods-dfv.js` and it made sense to separate that out.

## Testing instructions



1. Use forms as normal
2. Use admin forms as normal
3. You'll notice if it breaks immediately from JS errors etc
